File: Ordenamiento/SelectionSort.java
public class SelectionSort{

	int i,j,n,minimo,aux;
	int[] arr = {50,17,48,23,8,60};

	public static void main(String[] args){
		SelectionSort ss = new SelectionSort();
		ss.selectionSort(ss.arr); //Ordenamiento SelectionSort

		for(ss.i=0;ss.i<=ss.n-1;ss.i++){
			System.out.println(ss.arr[ss.i]);
		}
	}

	public void selectionSort(int[] A){
		n=A.length;
		for(i=0;i<=n-1;i++){
			minimo = i;
			for(j=i+1;j<=n-1;j++){
				if(A[j]<A[minimo]){
					minimo=j;
				}
			}
			//Intercambio
			aux=A[minimo];
			A[minimo]=A[i];
			A[i]=aux;
		}
	}

}

File: EDAII x Python/Ordenamiento/SelectionSort.py
class SelectionSort:

	arr = []

	def __init__(self, arr):
		'Constructor del objeto.' 
		self.arr = arr

	def selectionSort(self):
		'Algoritmo de ordenamiento.'
		n = len(self.arr)

		for i in range(0,n):
			minimo = i
			for j in range(i+1,n):
				if self.arr[j] < self.arr[minimo]:
					minimo = j

			# Intercambio
			aux = self.arr[minimo]
			self.arr[minimo] = self.arr[i]
			self.arr[i] = aux

# Ejecutanto el objeto
programa = SelectionSort([50,17,48,23,8,60])
programa.selectionSort()
print(programa.arr)